### PR TITLE
replace all instances of 'ln' with 'lightning' in README.md

### PR DESCRIPTION
**Pull request by shea256**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 
 ## Installation
 
 * If necessary, install Go according to the installation instructions here: http://golang.org/doc/install. It is recommended to add `$GOPATH/bin` to your `PATH` at this point.
-* Run the following command to obtain and install lnd, lncli, lnshell and all dependencies:
+* Run the following command to obtain and install lightningd, lightningcli, lightningshell and all dependencies:
 
 ```
-go get -u -v github.com/lightningnetwork/lnd/...
+go get -u -v github.com/lightningnetwork/lightningd/...
```
Comment (Roasbeef) [inline README.md]: Changing this without changing the name of the repo will cause future `go get ...` calls to fail.

File: README.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
-This version of Lnd is not yet fully-operational, but a proof of concept on testnet-L will likely exist soon.  The projection is it will be operational before the necessary malleability soft-forks are into bitcoin mainnet (may be significantly before if there are any delays in mainnet malleability fix).
+This version of lightningd is not yet fully-operational, but a proof of concept on testnet-L will likely exist soon.  The projection is it will be operational before the necessary malleability soft-forks are into bitcoin mainnet (may be significantly before if there are any delays in mainnet malleability fix).
 
 Don't try to port it to mainnet or an altcoin and use it today!  No really.  Lightning transactions will be fast, but for now, please wait just a little bit.
 
 ## Installation
 
 * If necessary, install Go according to the installation instructions here: http://golang.org/doc/install. It is recommended to add `$GOPATH/bin` to your `PATH` at this point.
-* Run the following command to obtain and install lnd, lncli, lnshell and all dependencies:
+* Run the following command to obtain and install lightningd, lightningcli, lightningshell and all dependencies:
```
Comment (Roasbeef) [inline README.md]: Please revert this line, the packages are purposefully named.

File: README.md [hunk 3]
```diff
@@ -1,20 +1,20 @@
-# lnd - Lightning Network Daemon
+# Lightning Network Daemon
```
Comment (Roasbeef) [inline README.md]: `lnd` here should be kept, it's the name of the daemon. All other swaps below from `lnd` to `lightningd` should also be reverted.

**Comment by Roasbeef** [comment]
Hey @shea256, thanks for the PR! 

Changing the name of the project isn't as simple as changing a few lines in the README. As I pointed out in the comments above, changing the instructions for `go get ...` will break the installation command for users. In order to change the project name, while maintaining links circulating, we'll have to create a new repo, and edit the `README` of this repo pointing over to the new one. 

As this change may be a bit involved, I'm closing this PR for now. We'll handle the steps required to make a smooth switchover.

**Comment by shea256** [comment]
@Roasbeef I realize that, I'm not a noob :smile: 

My point was to demonstrate how the language could be much better and how it would improve the branding / developer aesthetics, so I made some quick edits to the README.md file.

I figured someone on the team would have to go through and make edits to variable names and such and multiple commits would be required for the update.

Also I don't see why you'll need to change the name of the repo. Are you having people install directly from GitHub? I'm not quite familiar with go but surely it must have a package manager.

And sounds good RE the switchover.

**Comment by shea256** [comment]
It seems you should be able to do this without creating a new repo, but I could be mistaken.

**Comment by Roasbeef** [comment]
Yep, Go has a package manager. Go source code stored locally in a special "[workspace](https://golang.org/doc/code.html#Organization)". Within this workspace, packages are partitioned (under $GOPATH/src/) according to which version source control software each project uses. So if we change the name, then all `go get ...` pointers, and the folder structure locally would need to change also. Users could either re-name everything, or just nuke then start over. 

A new repo would be required in order to avoid killing any active links to `lnd` on the net. Links to `github.com/LightningNetwork/lnd` would be broken since Github doesn't set up redirects if a repo name is changed.

**Comment by Roasbeef** [comment]
Actually, I'm wrong: [Github does in fact redirect old links](https://help.github.com/articles/renaming-a-repository/)

**Comment by shea256** [comment]
Yes of course, I knew that. I rename repos all the time.

**Pull request closed without merging**